激活函数/为什么激活函数？.md

```diff
@@ -19,25 +19,28 @@ $$\large \sigma = {{1}\over{1+e^{-y}}}$$
 
 ## Tanh
 $$\large \tanh = {{1 - e^{-y}}\over{1 + e^{-y}}}$$
-- 定义域$[-1,1]$
+- 定义域 $[-1,1]$
 - 导数最大值：1
 	- 饱和函数，取值有上限
 		- 遇到极限导数链时会导致梯度消失
 
 ## ReLU
-$$\text{ReLU}(x) = \begin{cases} 0 & \text{if } x < 0 \\ x & \text{if } x \geq 0 \end{cases}$$
+$$\text{ReLU}(x) = \begin{cases}
```

Comment:
make it a line

Reply:
multi line cases needs to be in multi line in github